docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -134,16 +134,22 @@ this pattern.
 property. The `@timestamp` field must be included in every document indexed to
 the data stream.
 
-* A <<date,`date`>> or <<date_nanos,`date_nanos`>> field mapping for the
-`@timestamp` field.
+The template can also contain:
+
+* An optional field mapping for the `@timestamp` field. Both the <<date,`date`>> and 
+<<date_nanos,`date_nanos`>> field data types are supported. If no mapping is specified,
+the <<date,`date`>> field data type is used by default.
```

Comment:
maybe include the mapping snippet or that a date field with default options is then used?